dir.props

```diff
@@ -1,18 +1,23 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Build Tools Version -->
+  <PropertyGroup>
+    <BuildToolsVersion>1.0.21-prerelease</BuildToolsVersion>
+  </PropertyGroup>
```

Comment:
This is just a place-holder value.  My plan was to update this after the changes have been checked in and a new package uploaded to the feed.